### PR TITLE
Add generatePDF() function

### PR DESCRIPTION
**Pull request by benogle**
This adds `client.generatePDF(payload, options)` to the client for the `/api/v1/generate-pdf` REST endpoint.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +115,62 @@ const { statusCode, data } = await anvilClient.fillPDF(pdfTemplateID, payload, o
...
+  "dataType": "buffer"
+}
+const anvilClient = new Anvil({ apiKey })
+const { statusCode, data } = await anvilClient.generatePDF(payload, options)
+
+// Be sure to write the file as raw bytes
+fs.writeFileSync('generated.pdf', data, { encoding: null })
+```
+
+* `pdfTemplateID` (String) - The id of your PDF template from the Anvil UI
```
Comment (newhouse) [inline README.md:161]: Not a thing with this one, right?

File: README.md [hunk 2]
```diff
@@ -108,6 +115,62 @@ const { statusCode, data } = await anvilClient.fillPDF(pdfTemplateID, payload, o
...
+* `payload` (Object) - The JSON data that will fill the PDF template
+  * `title` (String) - _optional_ Set the title encoded into the PDF document
+  * `data` (Array of Objects) - The data that generates the PDF. See [the docs](https://useanvil.com/docs/api/generate-pdf#supported-format-of-data) for all supported objects
```
Comment (newhouse) [inline README.md:164]: `the docs` link no worky. 404...is that a thing yet, or you have another PR that adds this to the marketing site?

Waiting for this, I presume?
https://github.com/anvilco/marketing-site/pull/153

**Review by newhouse — APPROVED** [review]
Something in there to fix, but approving for when you're done!

**Review by Winggo — APPROVED** [review]
Looks good to me~

File: README.md (hunk 2 above)
Comment (benogle) [reply README.md:164]: yep

**Comment by newhouse** [comment]
Published as `v2.5.0`

**Pull request merged**